Essentials/src/com/earth2me/essentials/craftbukkit/InventoryWorkaround.java

```diff
@@ -267,4 +263,22 @@ public static void setItemInOffHand(final Player p, final ItemStack item) {
             }
         }
     }
+
+    public static int clearItemInOffHand(final Player p, final ItemStack item) {
+        if (hasMainHandSupport == null || hasMainHandSupport) {
```

Comment:
`if` will always return `true`, therefore making no need for it (can also be shortened using `!`)